[zh] Sync a task: egress/wildcard-egress-hosts/ (#14663)
istio/istio.io: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

`wikipedia.org` 站点均有自己的主机名，譬如：英语和德语对应的主机分别为 `en.wikipedia.org`
和 `de.rikipedia.org`。您希望通过通用配置项开启所有 Wikipedia 站点的 Egress 流量，无需单独配置每个语种的站点。

{{< boilerplate gateway-api-gamma-support >}}

## 开始之前  {#before-you-begin}

*  安装 Istio，启用访问日志记录，并采用默认阻止出站流量策略。

{{< tabset category-name="config-api" >}}

{{< tab name="Istio API" category-value="istio-apis" >}}

{{< text bash >}}
$ istioctl install --set profile=demo --set meshConfig.outboundTrafficPolicy.mode=REGISTRY_ONLY
{{< /text >}}

{{< tip >}}
您可以在 `demo` 配置文件以外的 Istio 配置上运行此任务，

> 只要您确保 [部署 Istio Egress 网关](/zh/docs/tasks/traffic-management/egress/egress-gateway/#deploy-Istio-egress-gateway)。

Reviewer: ```suggestion
只要您确保[部署 Istio Egress 网关](/zh/docs/tasks/traffic-management/egress/egress-gateway/#deploy-Istio-egress-gateway)。
```

    $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
    {{< /text >}}

> ## 配置流向 Wildcard 主机的引导流量   {#configure-direct-traffic-to-a-wildcard-host}

Reviewer: ```suggestion
## 配置流向 Wildcard 主机的引导流量   {#configure-direct-traffic-to-a-wildcard-host}
```
“引导”从动词改成了名词，感觉原意变了

Author: Thanks, reverted.

    <title>Wikipedia – Die freie Enzyklopädie</title>
    {{< /text >}}

> ### 清理到 Wildcard 主机的引导流量  {#cleanup-direct-traffic-to-a-wildcard-host}

Reviewer: ```suggestion
### 清理将流量引导至 Wildcard 主机的规则  {#cleanup-direct-traffic-to-a-wildcard-host}
```

{{< text bash >}}
$ kubectl delete serviceentry wikipedia
{{< /text >}}

> ## 配置到 Wildcard 主机的 Egress 网关流量 {#configure-egress-gateway-traffic-to-a-wildcard-host}

Reviewer: ```suggestion
## 配置到 Wildcard 主机的 Egress 网关流量规则 {#configure-egress-gateway-traffic-to-a-wildcard-host}
```
